integration_tests/models/source/bigquery/snowplow_ecommerce_events_stg.sql

```diff
@@ -0,0 +1,108 @@
+with prep as (
+select
+  *
+  except(contexts_com_snowplowanalytics_snowplow_web_page_1_0_0,
+    contexts_com_snowplowanalytics_snowplow_ecommerce_user_1_0_0,
+    contexts_com_snowplowanalytics_snowplow_ecommerce_page_1_0_0,
+    unstruct_event_com_snowplowanalytics_snowplow_ecommerce_snowplow_ecommerce_action_1_0_0,
+    contexts_com_snowplowanalytics_snowplow_ecommerce_cart_1_0_0,
+    contexts_com_snowplowanalytics_snowplow_ecommerce_product_1_0_0,
+    contexts_com_snowplowanalytics_snowplow_ecommerce_checkout_step_1_0_0,
+    contexts_com_snowplowanalytics_snowplow_ecommerce_transaction_1_0_0
+  ),
+  JSON_EXTRACT_ARRAY(contexts_com_snowplowanalytics_snowplow_web_page_1_0_0) AS contexts_com_snowplowanalytics_snowplow_web_page_1_0_0,
+  JSON_EXTRACT_ARRAY(contexts_com_snowplowanalytics_snowplow_ecommerce_user_1_0_0) as contexts_com_snowplowanalytics_snowplow_ecommerce_user_1_0_0,
+  JSON_EXTRACT_ARRAY(contexts_com_snowplowanalytics_snowplow_ecommerce_page_1_0_0) as contexts_com_snowplowanalytics_snowplow_ecommerce_page_1_0_0,
+  PARSE_JSON(unstruct_event_com_snowplowanalytics_snowplow_ecommerce_snowplow_ecommerce_action_1_0_0) as unstruct_event_com_snowplowanalytics_snowplow_ecommerce_snowplow_ecommerce_action_1_0_0,
+  JSON_EXTRACT_ARRAY(contexts_com_snowplowanalytics_snowplow_ecommerce_cart_1_0_0) as contexts_com_snowplowanalytics_snowplow_ecommerce_cart_1_0_0,
+  JSON_EXTRACT_ARRAY(contexts_com_snowplowanalytics_snowplow_ecommerce_product_1_0_0) as contexts_com_snowplowanalytics_snowplow_ecommerce_product_1_0_0,
+  JSON_EXTRACT_ARRAY(contexts_com_snowplowanalytics_snowplow_ecommerce_checkout_step_1_0_0) as contexts_com_snowplowanalytics_snowplow_ecommerce_checkout_step_1_0_0,
+  JSON_EXTRACT_ARRAY(contexts_com_snowplowanalytics_snowplow_ecommerce_transaction_1_0_0) as contexts_com_snowplowanalytics_snowplow_ecommerce_transaction_1_0_0
```

Comment:
As an easy way to ensure that the column prefix is tested, could you make one of these (in the input, and the processing) a `1_0_1` or something like that - that way we can just be certain that we're not hardcoding `1_0_0` somewhere in the models?